metadatastore/commands.py

```diff
@@ -1,34 +1,94 @@
 from __future__ import (absolute_import, division, print_function,
                         unicode_literals)
+
 import six
 from functools import wraps
 from itertools import count
-from .odm_templates import (RunStart, BeamlineConfig, RunStop,
-                            EventDescriptor, Event, DataKey, ALIAS)
-from .document import Document
+
 import datetime
 import logging
-from metadatastore import conf
-from mongoengine import connect,  ReferenceField
-import mongoengine.connection
 
-import datetime
+import boltons.cacheutils
 import pytz
 
-import uuid
-from bson import ObjectId
+from bson.dbref import DBRef
+
+from mongoengine import connect
+import mongoengine.connection
+
+from . import conf
+from .odm_templates import (RunStart, RunStop, EventDescriptor, Event, DataKey,
+                            ALIAS)
+from . import doc
+
 
 logger = logging.getLogger(__name__)
 
 
-__all__ = ['insert_beamline_config', 'insert_run_start', 'insert_event',
-           'insert_run_stop', 'insert_event_descriptor', 'find_run_stops',
-           'find_beamline_configs', 'find_event_descriptors', 'find_last',
-           'find_events', 'find_run_starts', 'db_connect', 'db_disconnect',
-           'format_data_keys', 'format_events', 'reorganize_event']
+# process local caches of 'header' documents these are storing object indexed
+# on ObjectId because that is what the reference fields in mongo are
+# implemented as.   Should move to uids asap
```

Comment:
Just some minor grammar stuff

```
# process local caches of 'header' documents. These caches are storing objects indexed
# on ObjectId because that is how the reference fields are implemented in mongo
# Should move to uids asap
```